docs/introduction/console-commands-for-application-management-phing-targets.md

```diff
@@ -220,10 +220,24 @@ Great tool when you want to translate your application into another language.
 
 For more information about translations, see [the separate article](/docs/introduction/translations.md).
 
-## Customize Phing properties (paths etc.)
-You can customize any property defined in `build.xml` via configuration file `build/build.local.properties` (use `build/build.local.properties.dist` as a template).
+## Customization of Phing targets and properties
+<!-- TODO: Remove or update the note below -->
```

Comment:
I've put it there so that the note gets removed in one of the following releases. Maybe it makes more sense to let it there, as people may use <=7.2.x for a while...

Reply:
ok my point was because in releaser there is one more dash
```suggestion
<!--- TODO: Remove or update the note below -->
```

Reply:
```php
private const TODO_PLACEHOLDER = '<!--- TODO';
```